ext/json/php_json.h

```diff
@@ -84,9 +84,13 @@ typedef enum {
 /* default depth */
 #define PHP_JSON_PARSER_DEFAULT_DEPTH 512
 
+/* default indent */
+#define PHP_JSON_ENCODER_DEFAULT_INDENT 4
+
 ZEND_BEGIN_MODULE_GLOBALS(json)
 	int encoder_depth;
 	int encode_max_depth;
+	size_t encoder_indent;
```

Comment:
what's the point of this?

Reply:
I ended up not needing that, removed the global :)